lib/services/android-project-service.ts

```diff
@@ -181,35 +184,37 @@ class AndroidProjectService implements IPlatformProjectService {
 	public isPlatformPrepared(projectRoot: string): IFuture<boolean> {
 		return this.$fs.exists(path.join(projectRoot, "assets", constants.APP_FOLDER_NAME));
 	}
-
-	private parseProjectProperties(projDir: string, destDir: string): void {
-		let projProp = path.join(projDir, "project.properties");
-
-		if (!this.$fs.exists(projProp).wait()) {
-			this.$logger.warn("Warning: File %s does not exist", projProp);
-			return;
-		}
-
-		let lines = this.$fs.readText(projProp, "utf-8").wait().split(os.EOL);
-
-		let regEx = /android\.library\.reference\.(\d+)=(.*)/;
-		lines.forEach(elem => {
-			let match = elem.match(regEx);
-			if (match) {
-				let libRef: ILibRef = { idx: parseInt(match[1]), path: match[2].trim() };
-				libRef.adjustedPath = this.$fs.isRelativePath(libRef.path) ? path.join(projDir, libRef.path) : libRef.path;
-				this.parseProjectProperties(libRef.adjustedPath, destDir);
+	
+	private parseProjectProperties(projDir: string, destDir: string): IFuture<void> {
+		return (() => {
+			let projProp = path.join(projDir, "project.properties");
+	
+			if (!this.$fs.exists(projProp).wait()) {
+				this.$logger.warn("Warning: File %s does not exist", projProp);
```

Comment:
Can we improve this warning? It would be great if we specify what scenarios are not working when the file is missing.